pkg/config/config.go

```diff
@@ -846,6 +846,7 @@ func InitConfig(config Config) {
 
 	// Serverless Agent
 	config.BindEnvAndSetDefault("serverless.logs_enabled", true)
+	config.BindEnvAndSetDefault("enhanced_metrics", true)
```

Comment:
We should consider making this `serverless.enhanced_metrics`. If I understand correctly, this will allow us to configure using the environment variable `DD_SERVERLESS_ENHANCED_METRICS` but also the configuration file like so:

```
serverless:
   logs_enabled: false
   enhanced_metrics: true
```

The downside is that the environment variable would be different from what is used in the Lambda Library :/

What are your thoughts?

Reply:
Is it possible to respect both values? And give DD_SERVERLESS_ENHANCED_METRICS higher precedence?

Reply:
✅